spec/models/user_spec.rb

```diff
@@ -1,24 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  let(:stem_credentials) do
-    OmniAuth::AuthHash.new(expires: true,
-                           expires_at: Time.zone.now.to_i,
-                           refresh_token: '123-refresh-token',
-                           token: '123-token')
-  end
-  let(:stem_info) do
-    OmniAuth::AuthHash::InfoHash.new(achiever_contact_no: '123456',
-                                     email: 'user@example.com',
-                                     first_name: 'Jane',
-                                     last_name: 'Doe')
-  end
-  let(:uid) { '987654321abcDEF' }
-  let(:user) do
-    User.from_auth(uid,
-                   stem_credentials,
-                   stem_info)
-  end
+  
+  let(:user) { create(:user, stem_user_id: "id-stem_user", email: 'user@example.com')}
```

Comment:
the idea of this test was the check OmniAuth was doing the correct thing with the data, by using the user factory instead we've lost that coverage, as with the test above, I think that using a proper uuid for the `achiever_contact_no` will fix whatever errors you were seeing